public/css/styles.css

```diff
@@ -279,38 +279,38 @@ body {
   align-items: center;
 }
 
-.DeveloperOptionsView_backdrop {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-
-  background-color: rgba(46, 48, 51, 0.38);
-}
-
-.DeveloperOptionsView_modal {
-  z-index: 1;
-  overflow-y: auto;
+.ModalView_modal {
   width: 100%;
   max-width: 500px;
   max-height: 80%;
-  margin-left: 10px;
-  margin-right: 10px;
```

Comment:
If we leave the `margin-left`/`margin-right` set, it makes the `dialog` not positioned in the middle of the screen.

#### Actual:

![](https://user-images.githubusercontent.com/558581/196594001-3e0bb973-779e-4f16-baf1-0ff16578f869.png)

#### Expected:

![](https://user-images.githubusercontent.com/558581/196594074-d772bd1e-1329-40c7-940a-67b218bdf31f.png)


But these were useful to make the modal not butt up against the very edge of the screen. It's not the end of the world though 🤔

![](https://user-images.githubusercontent.com/558581/196594117-b34427cf-e2a0-44a3-8873-f2171917f487.png)

Reply:
It looks like we can get the same effect by using `max-width: min(500px, 100% - 20px);` ✅

![](https://user-images.githubusercontent.com/558581/196594840-96787b40-9d69-4137-8883-8cfb0d576527.gif)